appveyor.yml

```diff
@@ -0,0 +1,22 @@
+environment:
+  matrix:
+    - nodejs_version: "4"
+    - nodejs_version: "6"
+
+install:
+  - ps: Install-Product node $env:nodejs_version x64
+  - npm i -g npm@latest
+  - node --version
+  - npm --version
+  - npm install
+
+test_script:
+  - npm run build
+  # Hax - Some snapshot tests don't currently work on Windows, so the tests are
```

Comment:
I need the hack for now, otherwise the build fails :( I'll remove it at
some point.

Sent from my phone.

On Sep 2, 2016 12:50 PM, "Christoph Pojer" <notifications@github.com> wrote:

> In appveyor.yml
> <https://github.com/facebook/jest/pull/1465#discussion_r77402861>:
>
> > @@ -0,0 +1,22 @@
> > +environment:
> > +  matrix:
> > +    - nodejs_version: "4"
> > +    - nodejs_version: "6"
> > +
> > +install:
> > +  - ps: Install-Product node $env:nodejs_version x64
> > +  - npm i -g npm@latest
> > +  - node --version
> > +  - npm --version
> > +  - npm install
> > +
> > +test_script:
> > +  - npm run build
> > +  # Hax - Some snapshot tests don't currently work on Windows, so the tests are
>
> Maybe remove this Hax :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/jest/pull/1465/files/a83dddfad34dbc11385a6e1606ef3ff4b84c980e#r77402861>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFnHfNi2eFOaFwXervZLrsQXxJ5I_Rkks5qmH4VgaJpZM4JoL_3>
> .
>